runtime/compiler/build/toolcfg/gnu/common.mk

```diff
@@ -104,8 +104,14 @@ CX_FLAGS+=\
     -Wreturn-type \
     -fno-strict-aliasing
 
+ifeq ($(JITSERVER_SUPPORT),)
+    CXX_FLAGS+=\
+        -std=c++0x
+else
+    CXX_FLAGS+=\
+        -std=c++11
```

Comment:
I wonder whether this is going to be an obstacle for eliminating the #ifdef JITSERVER_SUPPORT in the end.